GitUI/MessageBoxes.cs

```diff
@@ -45,9 +51,14 @@ internal MessageBoxes()
 
         private static MessageBoxes Instance => instance ?? (instance = new MessageBoxes());
 
+        public static void ShowErrorMessageBox([CanBeNull] IWin32Window owner, string text, string caption = null)
+        {
+            MessageBox.Show(owner, text, caption ?? Strings.Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
+        }
```

Comment:
```suggestion
        public static void ShowErrorMessageBox([CanBeNull] IWin32Window owner, string text, string caption = null)
            => MessageBox.Show(owner, text, caption ?? Strings.Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
```
This applies to the changed and new functons below, too.